doc/releases/nautilus.rst

```diff
@@ -0,0 +1,516 @@
+v14.1.0 Nautilus
+================
+
+.. note: These are draft notes for the first Nautilus release.
+
+This is the first stable release of Nautilus.
+
+Major Changes from Mimic
+------------------------
+
+- *Dashboard*:
+
+
+- *RADOS*:
+
+  * The new *msgr2* wire protocol brings support for encryption on the wire.
+  * The number of placement groups (PGs) per pool can now be decreased
+    at any time, and the cluster can automatically tune the PG count
+    based on cluster utilization or administrator hints.
+  * Physical storage devices consumed by OSD and Monitor daemons are
+    now tracked by the cluster along with health metrics (i.e.,
+    SMART), and the cluster can apply a pre-trained prediction model
+    or a cloud-based prediction service to warn about expected
+    HDD or SSD failures.
+
+- *RGW*:
+
+  * S3 lifecycle transition for tiering between storage classes.
+  * A new web frontend (Beast) has replaced civetweb as the default,
+    improving overall performance.
+  * A new publish/subscribe infrastructure allows RGW to feed events
+    to serverless frameworks like knative or data pipelies like Kafka.
+  * A range of authentication features, including STS federation using
+    OAuth2 and OpenID::connect and an OPA (Open Policy Agent)
+    authentication delegation prototype.
+  * The new archive zone federation feature enables full preservation
+    of all objects (including history) in a separate zone.
+
+- *CephFS*:
+
+
+- *RBD*:
+
+
+- *Misc*:
+
+
+Upgrading from Mimic or Nautilus
+--------------------------------
+
+Notes
+~~~~~
+
+* During the upgrade from Luminous to nautilus, it will not be
+  possible to create a new OSD using a Luminous ceph-osd daemon after
+  the monitors have been upgraded to Nautilus.  We recommend you avoid adding
+  or replacing any OSDs while the upgrade is in process.
+
+* We recommend you avoid creating any RADOS pools while the upgrade is
+  in process.
+
+* You can monitor the progress of your upgrade at each stage with the
+  ``ceph versions`` command, which will tell you what ceph version(s) are
+  running for each type of daemon.
+
+Instructions
+~~~~~~~~~~~~
+
+#. If your cluster was originally installed with a version prior to
+   Luminous, ensure that it has completed at least one full scrub of
+   all PGs while running Luminous.  Failure to do so will cause your
+   monitor daemons to refuse to join the quorum on start, leaving them
+   non-functional.
+
+   If you are unsure whether or not your Luminous cluster has
+   completed a full scrub of all PGs, you can check your cluster's
+   state by running::
+
+     # ceph osd dump | grep ^flags
+
+   In order to be able to proceed to Mimic, your OSD map must include
+   the ``recovery_deletes`` and ``purged_snapdirs`` flags.
```

Comment:
@liewegas let's mention that the flag `pglog_hardlimit` will be on by default on fresh nautilus installs